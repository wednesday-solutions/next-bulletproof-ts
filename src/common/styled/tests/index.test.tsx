import React from "react";
import { render } from "@testing-library/react";
import { Container } from "../index";

describe("<Container/>", () => {
  const props = {
    maxwidth: 10,
    padding: 10,
  };

  it("should ensure it match the snapshot", () => {
    const { baseElement } = render(<Container {...props} />);
    expect(baseElement).toMatchSnapshot();
  });
});
