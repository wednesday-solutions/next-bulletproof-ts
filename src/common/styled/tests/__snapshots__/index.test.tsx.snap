// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container/> should ensure it match the snapshot 1`] = `
.emotion-0.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  max-width: 10px;
  width: 100%;
  margin: 0 auto;
  padding: 10px;
}

<body>
  <div>
    <div
      class="emotion-0"
    />
  </div>
</body>
`;
